priv/repo/migrations/20201110152910_create_notifications.exs

```diff
@@ -29,6 +33,7 @@ defmodule Skate.Repo.Migrations.StoreNotifications do
   end
 
   def down do
+    execute("DROP TYPE notification_reason")
```

Comment:
Should this maybe be in the opposite order (drop the table using the type, _then_ the type)? Or does Ecto already automagically reverse the order of the steps in `down`?